core_backend/app/contents/models.py

```diff
@@ -98,9 +109,16 @@ async def save_content_to_db(
     content: ContentCreate,
     asession: AsyncSession,
 ) -> ContentDB:
+    """Vectorize the content and save to the database.
+
+    :param user_id: The ID of the user requesting the save.
```

Comment:
Can we please use numpy style docstrings? That's the standard we use.
Also, we used to be very particular about docstrings but since we started using typing, have found less use for it. Having said that, please add wherever you find the code to not be intuitive

Reply:
1. can definitely switch to numpy style docstrings.
2. for the most part, the docstrings would not be useful for active/main developers. it would be for users, new developers, and long-term maintenance.